### PR TITLE
use display name in download activity

### PR DESCRIPTION
**Pull request by szaimen**
Should close https://github.com/nextcloud/files_downloadactivity/issues/85

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: lib/CurrentUser.php [hunk 1]
```diff
@@ -85,7 +85,7 @@ public function getUID(): ?string {
 		if ($this->sessionUser === false) {
 			$user = $this->userSession->getUser();
 			if ($user instanceof IUser) {
-				$this->sessionUser = (string) $user->getUID();
+				$this->sessionUser = (string) $user->getDisplayName();
```
Comment (nickvergessen) [inline lib/CurrentUser.php:88]: the function is called `getUID`
What is actually missing is a translation of the detail in the activity afterwards when it is rendered?
We shouldn't write the display name into the activity.
But then again this works fine here.

Comment (nickvergessen) [reply lib/CurrentUser.php:88]: Bildschirmfoto von 2022-02-23 16-35-16

Comment (nickvergessen) [reply lib/CurrentUser.php:88]: User handling and getting the display name is actually here:
https://github.com/nextcloud/files_downloadactivity/blob/2482f99fa20602061e985e40f965427f3f7cbd67/lib/Activity/Provider.php#L230-L253

Comment (nickvergessen) [reply lib/CurrentUser.php:88]: The bug actually is from the trackdownload app which is also adding the IP address (not as a second parameter) to the userId: 
https://github.com/TimoKunze/files_trackdownloads/blob/master/lib/Activity/Listener.php#L113

**Comment by szaimen** [comment]
Thanks for the analysis, Joas! :)

**Pull request closed without merging**